pkg/cable/libreswan/libreswan.go

```diff
@@ -186,6 +187,45 @@ func extractSubnets(endpoint subv1.EndpointSpec) []string {
 	return subnets
 }
 
+const (
+	ignoreExitError  = true
+	dontIgnoreErrors = false
+)
+
+func whack(ignoreOnExitError bool, args ...string) error {
+
+	var err error
+	for i := 0; i < 3; i++ {
+		cmd := exec.Command("/usr/libexec/ipsec/whack", args...)
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		klog.V(log.TRACE).Infof("Whacking with %v", args)
+
+		if err = cmd.Run(); err == nil {
+			break
+		}
+
+		klog.Errorf("error %v whacking with args: %v", err, args)
+		time.Sleep(1 * time.Second)
+
+	}
+
+	if err != nil {
+		switch err2 := err.(type) {
+		case *exec.ExitError:
+			if ignoreOnExitError {
+				klog.Errorf("error adding a connection with args %v; got exit code %d: %v", args, err2.ExitCode(), err)
```

Comment:
AFAICT, idiomatic Go is to reuse the variable name:

```suggestion
		switch err := err.(type) {
		case *exec.ExitError:
			if ignoreOnExitError {
				klog.Errorf("error adding a connection with args %v; got exit code %d: %v", args, err.ExitCode(), err)
```

Reply:
Oh, I thought it was clobbering err, but it's just moving it into it's right type.